flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -1274,6 +1274,8 @@ public void alterTable(
      * @param objectIdentifier The fully qualified path of the table to drop.
      * @param ignoreIfNotExists If false exception will be thrown if the table to drop does not
      *     exist.
+     * @return true if table was dropped, it will return false if table was not found and ignore if
```

Comment:
difficult to parse the sentence I suggest using the flag name ignoreIfNotExists

Reply:
rephrased it